lib/shared/bucketing-assembly-script/__tests__/bucketing/segmentation.test.ts

```diff
@@ -394,12 +394,52 @@ describe('SegmentationManager Unit Test', () => {
             })
         })
 
+        describe('evaluateOperator should handle audienceMatch filter', () => {
+            it('should work for an AND operator', () => {
+                const filters = [
+                    { type: 'user', subType: 'country', comparator: '=', values: ['Canada'] },
+                    { type: 'user', subType: 'email', comparator: '=', values: ['dexter@smells.nice', 'brooks@big.lunch'] },
+                    { type: 'user', subType: 'appVersion', comparator: '>', values: ['1.0.0'] }
+                ]
+
+                const operator = {
+                    filters,
+                    operator: 'and'
+                }
+
+                const data = {
+                    country: 'Canada',
+                    email: 'brooks@big.lunch',
+                    platformVersion: '2.0.0',
+                    appVersion: '2.0.2',
+                    platform: 'iOS'
+                }
+
+                const audienceMatchOperator = {
+                    filters: [
+                        {
+                          type: 'audienceMatch',
+                            comparator: '=',
+                          _audiences: [
+                              {
```

Comment:
these arent going to be populated in the config so you should test it when its just a string id

Reply:
oh unless youre populating it before calling evaluateOperator

Reply:
I was originally which is why it was this way in the tests. Its changed now though